ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -998,6 +995,12 @@ private command revCopyMobileFiles pFiles, pBaseFolder, pAppBundle, pAppTarget,
          next repeat
       end if
       
+      -- If the file is an entitlement then handle appropriately
+      if tSource ends with ".xcent" then
+         put url ("binfile:" & tSource) after rCustomEntitlements
```

Comment:
Makes sense! If `textDecode` could validate then I'd suggest doing that for a sanity check but since it doesn't validate I think the "just treat as binary data" approach makes good sense.